src/icons/BorderStyleDashed.tsx

```diff
@@ -0,0 +1,16 @@
+import Box from "@mui/material/Box";
+import { MENU_BUTTON_FONT_SIZE_DEFAULT } from "../controls/MenuButton";
+
+function BorderStyleDashed() {
+  return (
+    <Box
```

Comment:
Creative use of `Box` to achieve more useful icons for these border styles. One caveat unfortunately is that these border "icons" won't have the same props as the other icons which are stand MUI `SvgIcon`s. For instance,  won't support the `color` or `fontSize` props. That might be okay, but it's a quirk that could be confusing to users.